.github/ISSUE_TEMPLATE/rule-request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Rule request
+about: Requesting a new rule
+title: ''
+labels: rule requests
+assignees: ''
+
+---
+
+**Add the name of the rule you'd like to add**
+Add rule name here. Prevent/Ensure ...
+
+**Describe the rule**
+A clear description in a single sentence
+
+**What triggers the rule**
+Add a detailed description of what behaviors should trigger this rule. This will help us best understand how to implement it
+
+ **Failure message should the rule fail**
+Short message for when this rule fails
+
+**What SCMs is this rule eligible for**
+Add here one or more: Github, Gitlab, etc.
```

Comment:
How about including "Local" as well?

Reply:
Right, adding

Reply:
Local is not exactly parallel to `Github` or `Gitlab`
I think running on local git repo is a behavior some rules should support, and it will depend on how the rule is written (and I think we should run all `Github` and `Gitlab` logic on all of them with OR functionality between them

Reply:
Agree, so there should be a note regarding the rule relevancy for local validation.
For instance, a rule that validates something based upon the metadata provided from github can't run locally.

Reply:
@idanshahar 
what do you suggest?
note that asking **Should this rule run on local** or something similar is not very intuitive and users might not fully understand what that means

also, I agree we can't validate rules that require GitHub metadata but can we expect people that open requests for a new rules to know what information is related to github specifically and what is not?

Note: we can technically skip rules that should not run on local without any external information about the rule

Reply:
Although we can technically skip rules that are not relevant for a local run, I think we should add a disclaimer that will make people think about the local scenario when they open a new rule request. For example - ensuring there is at least one SCA in a repo. How do we define a repo in a local validation? 
I'm thinking of something like that:
"Does this rule relevant when running allero validate on a local filesystem? If so, is there any different expected behavior for a local run"?

Reply:
Cool. Adding